devtools/prod-envs/qcarchive-worker-openff-ani.yaml

```diff
@@ -0,0 +1,22 @@
+name: qcarchive-worker-openff-ani
+channel_priority: flexible
+channels:
+  - conda-forge
+  - pytorch-nightly
+  - defaults
+dependencies:
+  - python =3.7
+  - qcfractal
+  - qcengine =0.15.0
```

Comment:
This needs to be pinned to the next release version to support ANI2x.

Reply:
Done!